packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.h

```diff
@@ -545,6 +545,10 @@ namespace Ioss {
       }
     }
 
+    virtual std::vector<size_t> get_all_block_connectivity(const std::string &field_name, void *data, size_t data_size) const;
```

Comment:
Yeah, this references the general comment that I made below in regards to the original design that is centered around element blocks. Generalization to other grouping entities would probably necessitate a naming convention change

My initial work actually focused on just the parallel database and when I realized that I could extend it to the serial case by simply filling in the output data piece-wise per block, I ended up with the current code which allows you to simply pass in a field name. I suppose it might be possible to modify the parallel version so that if the field name is "connectivity" or "connectivity_raw", then it goes through the connectivity code. This way, we could remove `get_all_block_connectivity` as a user facing function and have `get_all_block_field_data` as the only public API